README.md

```diff
@@ -840,6 +841,95 @@ public class Handler : IRequiresAsyncInitialization
+
+If you obtain an `Owned<T>`, you access the `T` instance using the `Value` property, and you dispose the `Owned<T>` instance to release the `T` instance back to the container. The container will never dispose the contained `T` instance automatically, not even when the container itself is disposed. It’s now your responsibility to call `Owned<T>.Dispose()` when you’re ready. If you don’t, the `T` instance and all its dependencies will never be cleaned up.
+
+The typical scenario is when you combine this with delegates by injecting `Func<Owned<Y>>` or `Func<X, Owned<Y>>` into a component. This enables the component to acquire and release `Y` instances as many times as it needs, possibly with varying parameters, and possibly overlapping with each other before releasing them back to the container. Here’s one such usage:
+
+```cs
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using StrongInject;
+using System;
+using System.Threading.Tasks;
+
+[Register(typeof(SomeViewModel))]
+[Register(typeof(SomeDbContext))]
+public partial class Container : IContainer<SomeViewModel>
+{
+    [Factory]
+    public static SqlConnection CreateSqlConnection()
+    {
+        throw new NotImplementedException();
+    }
+}
+
+public class SomeViewModel
+{
+    private readonly Func<Owned<SomeDbContext>> dbContextFactory;
+
+    public SomeViewModel(Func<Owned<SomeDbContext>> dbContextFactory)
+    {
+        this.dbContextFactory = dbContextFactory;
+    }
+
+    public async Task SaveAsync(string newName)
+    {
+        // Using Func<Owned<SomeDbContext>> instead of Func<SomeDbContext> ensures that the context will not be
+        // disposed while this method is still using it, even if SomeViewModel is released back to the container.
+
+        // Using Func<Owned<SomeDbContext>> instead of Owned<SomeDbContext> gives the ability for SaveAsync to
+        // get a new SomeDbContext instance that is not being used anywhere else, so overlapping calls to SaveAsync
+        // are no problem.
+
+        // By modeling the dependency this way, SomeViewModel remains agnostic about whether SomeDbContext is
+        // outliving it by being pooled and recycled (not shown here). Both SomeViewModel and SomeDbContext are
+        // even agnostic about whether the SqlConnection is being pooled and recycled.
+
+        // Each component is agnostic about how its dependencies are created and cleaned up. This allows decisions
+        // to be changed in the configuration of the components without breaking them.
+        // This is naturally the case already when injecting a dependency directly the way SqlConnection is injected
+        // into SomeDbContext; what’s special here is showing how to achieve the same decoupling when the same
```

Comment:
> what’s special here is showing how to achieve the same decoupling when the same

I think one of the key points here is this stops each SqlConnection living as long as the ViewModel, preventing resource leaks.

Reply:
That's comparing `Func<Owned<SqlConnection>>` to `Func<SqlConnection>`, but I meant to be comparing `Func<Owned<SqlConnection>>` to `SqlConnection`.

The immediate response to "resource leaks" is "but I can dispose the SQL connection I got from `Func<SqlConnection>`, so it's not the best example of why you want `Owned`.

Reply:
Scratch that, I'm mixed up. This is `DbContext` and it's a good example.

Reply:
I think the default people are used to is to have the DbContext own the connection and dispose it when the DbContext is disposed, so there's an "easy workaround" for using `Func<Owned<T>>`, but I didn't have a better idea yet so I'm ignoring this.

Reply:
Pushed updates. Do you think I should still add more about this here, given the clarifications?

Reply:
I added a paragraph about this outside the code sample.